babel.config.js

```diff
@@ -382,3 +383,41 @@ function pluginPackageJsonMacro({ types: t }) {
     },
   };
 }
+
+// Match the Node.js behavior (the default import is module.exports)
+function pluginDefaultImportNode({ types: t }) {
```

Comment:
This plugin transforms `import foo from "mod"` into `const foo = require("mod")`, which is what Node.js does.